nemo_skills/inference/eval/bfcl.py

```
@@ -205,13 +213,16 @@ def parse_output_dict(self, output_dict: dict):
         tool_calls = [] if output_dict["message"].tool_calls is None else output_dict["message"].tool_calls
 
         try:
+            tool_calls = output_dict["message"].tool_calls
             generation = [{func_call.function.name: func_call.function.arguments} for func_call in tool_calls]
             tool_call_ids = [func_call.id for func_call in tool_calls]
         except Exception:
```

Ruff (0.14.0)

Do not catch blind exception: `Exception`

(BLE001)